docs/variants/asus_kgpe_d16/dasharo-compatibility/generic-test-setup.md

```diff
@@ -0,0 +1,19 @@
+# Dasharo Compatibility: generic test setup
+
+## Test setup
+
+Test setup is a set of procedures to be executed before the test execution.
+Typically, the same setup can be reused by multiple test cases, so there is no
+need to execute the setup actions before each independent case.
+
+### Generic test setup
+
+#### Firmware
+
+1. Obtain `FIRMWARE` binary
+    1. you can download it from [releases](../releases.md)
+       section
+    1. or you can build one yourself as shown in the
+       [Building manual](../building-manual.md) section
+1. Flash `FIRMWARE` binary to the DUT
```

Comment:
Problem z numeracją.

Reply:
The numbering of the points is done automatically.

Reply:
Keep using English in public repos please @kl3mdeb